brage-import/src/test/java/no/sikt/nva/brage/migration/record/ProjectTest.java

```diff
@@ -0,0 +1,77 @@
+package no.sikt.nva.brage.migration.record;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import java.net.URI;
+import java.util.Map;
+import no.unit.nva.model.funding.FundingBuilder;
+import nva.commons.core.Environment;
+import nva.commons.core.paths.UriWrapper;
+import org.junit.jupiter.api.Test;
+
+class ProjectTest {
+
+    @Test
+    void shouldCreateVerifiedFundingWhenNfrFunding() {
```

Comment:
`WhenFundingSourceIsNfr` ?

Reply:
Other than this, the naming of the tests makes it very clear what is going on, nice.